src/styles/reference.scss

```diff
@@ -0,0 +1,168 @@
+.reference-page {
+  table {
+    width: 100%;
+  }
+
+  h2,
+  h3 {
+    margin-top: 20px;
+    margin-bottom: 10px;
+  }
+}
+
+.api-modal-container {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(0, 0, 0, 0.5);
+  align-items: center;
+  justify-content: center;
+  z-index: 99999;
+
+  &--open {
+    display: flex;
+  }
+}
+
+.api-modal {
+  width: 800px;
+  max-width: 90vw;
+  max-height: 90vh;
+  border-radius: var(--amplify-radii-medium);
+}
+
+.api-model__header {
+  align-items: flex-start;
+}
+
+.api-modal__breadcrumbs {
+  flex: 1 0 0;
+  padding-bottom: var(--amplify-space-medium);
+  margin-bottom: var(--amplify-space-xs);
+  overflow: scroll;
```

Comment:
I think if we set the overflow to `auto` it could hide the empty scrollbars from the element, but still allow for scrolling?